doc/architecture/SENSORFUSION.md

```diff
@@ -47,11 +47,11 @@ The application uses the same [State Machine](https://github.com/BlueAndi/RadonU
 
 ### HAL
 Some changes have been made to the HAL.
-![HALSensorFusion](http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/BlueAndi/RadonUlzer/SensorFusion/doc/architecture/uml/LogicalView/SensorFusion/HAL_SensorFusion.puml)
+![HALSensorFusion](http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/BlueAndi/RadonUlzer/SensorFusion/sync/doc/architecture/uml/LogicalView/SensorFusion/HAL_SensorFusion.puml)
```

Comment:
This is not necessary. What happens when the sync branch gets deleted? You could use SF/base. and change it before merging to main. Or use the main branch directly, but it may be broken until the merge is done.